app/AppKernel.php

```diff
@@ -22,7 +22,7 @@ class AppKernel extends Kernel
      *
      * @return array
      */
-    public function registerBundles()
+    public function registerBundles(): array
```

Comment:
Good question, I think it is a bit too early for that. I tried looking for articles about it but it didn't turn up much